### PR TITLE
Update workflow steps

### LINKED ISSUE #7: Submodule update by GitHub Actions
Steps:

- Get & Use `gh` executable
- Login with token
- Fetch & Commit submodule changes locally
- Push commits & open PR in main repo

### PR DESCRIPTION
**Pull request by CloneWith**
Fixes #7.

### CODE REVIEW

**Comment by CloneWith** [comment]
Now we had made it possible to push changes to the remote, and we just need to fix the PR part.

```
pull request create failed: GraphQL: Resource not accessible by integration (createPullRequest)
```

**Comment by CloneWith** [comment]
OK, now worked👌

**Comment by NaughtyChas** [comment]
Successfully created an PR in parent repository, things should be working right now. 

Wondering if we can adjust the settings to display commit messages as initial PR details instead of fixed messages in this line:

`gh pr create --title "Update submodules" --body "This PR updates the submodules." --base main --head update-submodules`

**Comment by NaughtyChas** [comment]
The latest commit does its trick. PR available [here](https://github.com/osu-atri/osu-dictionary/pull/34).

If there's no other features or parameters to be added and adjusted, this PR can be merged right away.

**Comment by CloneWith** [comment]
Should we add a label to the newly added PR, to distinguish it from manually created PRs?
Like `#actions`, which I'll add later.

**Comment by NaughtyChas** [comment]
> Should we add a label to the newly added PR, to distinguish it from manually created PRs? Like `#actions`, which I'll add later.

Yes please.

**Comment by CloneWith** [comment]
I think the `@*` labeling part should be done individually in the main repo by a labeler, and I'll add the `#actions` label first.

File: .github/workflows/submoduleNotifyParent.yml [hunk 1]
```diff
@@ -55,12 +55,15 @@ jobs:
...
         run: |
           COMMIT_MESSAGES=$(git log --oneline origin/main..HEAD)
+          CHANGED_FILES=$(git diff --name-only origin/main..HEAD)
           echo "::set-output name=messages::$COMMIT_MESSAGES"
+          echo "::set-output name=files::$CHANGED_FILES"
 
       - name: Create Pull Request
         run: |
-          gh pr create --title "Update lazer-wiki" --body "${{ steps.commit_messages.outputs.messages }}" --base main --head update-submodules
+          BODY="Commit Messages:\n${{ steps.commit_messages.outputs.messages }}\n\nChanged Files:\n${{ steps.commit_messages.outputs.files }}\n\n*This PR is auto-generated by Github Actions due to an update of a submodule.*"
```
Comment (NaughtyChas) [inline .github/workflows/submoduleNotifyParent.yml:68]: Updated commit messages. Now the PR body should display:

- Autogenerated PR message: `This PR is auto-generated by Github Actions due to an update of a submodule`
- Incicent commit(s)
- Changed files

**Review by NaughtyChas — APPROVED** [review]
Looks good.
PR in parent repository has deleted, see how this script will do.

**Comment by NaughtyChas** [comment]
[Actions page](https://github.com/osu-atri/lazer-wiki/actions/runs/10070941899/job/27840921576)

Triggered action yml file is not in its latest version. @CloneWith

**Comment by CloneWith** [comment]
Got that, would merge first

**Comment by NaughtyChas** [comment]
Commit messages obviously needs a little bit of update. Push to `dev-moduleSync` branch for fix soon

**Comment by CloneWith** [comment]
> [PR Page](https://github.com/osu-atri/osu-dictionary/pull/37)
> 
> Commit messages obviously needs a little bit of update. Push to `dev-moduleSync` branch for fix soon

We can use HTML tag `<br>` for this

**Pull request merged**